From e8274041b5cfef6489081df5c6de93bccdf6c670 Mon Sep 17 00:00:00 2001
From: Antonio Rojas <arojas@archlinux.org>
Date: Tue, 3 Sep 2024 22:08:02 +0200
Subject: [PATCH] Fix build

communicator/src/communicator.cpp:11:10: fatal error: KEMailClientLauncherJob: No such file or directory
---
 src/CMakeLists.txt | 1 +
 1 file changed, 1 insertion(+)

diff --git a/src/CMakeLists.txt b/src/CMakeLists.txt
index c832175..a05fb0a 100644
--- a/src/CMakeLists.txt
+++ b/src/CMakeLists.txt
@@ -86,6 +86,7 @@ target_link_libraries(${PROJECT_NAME}
     KF6::I18n
     KF6::Service
     KF6::KIOCore
+    KF6::KIOGui
     # KF6::KIOFileWidgets
     #[[KF6::KIOWidgets]])
 
-- 
GitLab

